tests/template/create_template_test.go

```diff
@@ -0,0 +1,192 @@
+package template_test
+
+import (
+	"os"
+	"strings"
+
+	"github.com/nimblehq/gin-templates/tests"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = BeforeEach(func() {
+	tests.ChangeDirectory(TemplateGeneratedPath + "/test-gin-templates")
+})
+
+var _ = Describe("Create template", func() {
```

Comment:
To keep with the Go convention of keeping tests close to the source file, shouldn't this file be in `cmd` and named `create_test.go`?

Reply:
Yes it should but I think it is better to separate in this case because in the future we will have addon and variants and they are part of `create` command. It is difficult to manage the test suite (the `BeforeSuite` and `AfterSuite` hooks) if we decide to breaking the tests into each addon.

What do you think?

Reply:
The branch name should be a parameter in the command 

![image](https://user-images.githubusercontent.com/696529/108466356-13660700-72b6-11eb-84b5-836120255cad.png)

cookie-cutter supports specifying a branch (see [docs](https://cookiecutter.readthedocs.io/en/1.7.2/usage.html#works-directly-with-git-and-hg-mercurial-repos-too)):

![image](https://user-images.githubusercontent.com/696529/108466397-24167d00-72b6-11eb-8ce2-ccb531a2d4e8.png)

Reply:
Agree! Fixed in eb0eed87b0e69e7443d7c753a9898423a487a552

Reply:
Fixed in ce89d2babcc1db8397def10361acf21b66e633c4